### DISCUSSION TITLE
installed 7.1 computers os 7.1 but everything works as it should but internet

### DISCUSSION THREAD

**Discussion by Maierdan46** [elementary/triage#442] [Q&A]
Hello!
installed 7 computers os 7.1 but everything works as it should but internet is missing after updates ! What is the problem? will the same os 8.0? How to do this case?
Thank you if you answer honestly.
Hot
UTV

**Comment by Antonio476587** [comment]
Hello, I hope you are fine dude, I had the same problem as you.

I had installed Elementary OS 7.1 Horus, and it semmed to work fine. I ran the apt updates and those things, when I restarted the wifi stopped working.

What worked for me is the following:

1.- I followed the steps here https://www.reddit.com/r/elementaryos/comments/148zeiw/comment/jo426ho/ 
2.- After that I did what is marked here https://elementaryos.stackexchange.com/questions/29649/macbookpro-wifi/29660#29660

Now my wifi is working fine. But I'm having an issue:

``` bash
seo@Latitude-E5430-non-vPro-2b24c3cc:~$ rfkill list all
1: phy0: Wireless LAN
	Soft blocked: no
	Hard blocked: no
2: dell-wifi: Wireless LAN
	Soft blocked: no
	Hard blocked: no
```

I have two drivers.
